Patterns/Italian/Italian-DateTime.yaml

```diff
@@ -1,162 +1,206 @@
 ---
 TillRegex: !simpleRegex
-  def: (?<till>fino al|fino a|al|a|--|-|—|——|~)
-RangeConnectorRegex : !simpleRegex
-  def: (?<and>e|al|a|--|-|—|——)
+  def: (?<till>fino a(l(l[aoe'])?|gli|i)?|a(l(l[aoe'])?|gli|i)?|e (il?|l[aoe']|gli)|--|-|—|——|~)
```

Comment:
Is this version of the YAML file in sync with the more recent changes in English? Maybe new regexes or tweaks to them?